src/components/Match/MatchSummary.tsx

```diff
@@ -0,0 +1,55 @@
+import { Fragment } from 'react';
+import { MainReport, MatchSummary } from '../../types/common';
+import FormattedNumber from '../common/FormattedNumber';
+import MatchResultComponent from '../common/MatchResult';
+import MatchUserSummaryComponent from './MatchUserSummary';
+
+interface Props {
+    match: MatchSummary;
+    previousMatch?: MatchSummary;
+    userIDs: string[];
+    report: MainReport;
+}
+
+export default function MatchSummaryComponent({ match, userIDs, report, previousMatch }: Props) {
+    const date = new Date(match.CreatedAt);
+    const sessionDate = new Date(date.getTime());
+    sessionDate.setHours(sessionDate.getHours() - 8);
```

Comment:
What's the reasoning behind `- 8`?

Also kinda related... What does session mean in this context?

Reply:
The _gaming_ session is useful for grouping matches by day. The `-8` part just moves the _next day_ time to 8AM instead of 12AM.
This was moved to the `groupMatches` function in 5f1ca9ef976335739f5180e5c4851d4d43b26ca2.

Reply:
Clear!